src/components/Compendia/Explore.js

```diff
@@ -7,17 +7,15 @@ import { List } from 'components/shared/List'
 import { Row } from 'components/shared/Row'
 import { links } from 'config'
 
-const ListItem = ({ href, text }) => {
-  return (
-    <Box as="li" margin={{ bottom: 'small' }}>
-      <Anchor
-        href={href}
-        label={text}
-        onClick={() => gtag.trackExploredUsageClick(href)}
-      />
-    </Box>
-  )
-}
+const ListItem = ({ href, text }) => (
+  <Box as="li" margin={{ bottom: 'small' }}>
+    <Anchor
+      href={href}
+      label={text}
+      onClick={() => gtag.trackExploredUsageClick(href)}
+    />
+  </Box>
+)
```

Comment:
Since we no longer use `outboundClick`(which was removed in the latest `gtag` update), I've omitted the `return` statement here.